docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -277,7 +277,7 @@ to 300 seconds, use: `./mvnw verify -Pnative -Dquarkus.test.wait-time=300`.
 
 [WARNING]
 ====
-This procedure was formerly accomplished using the `@NativeImageTest` annotation. `@NativeImageTest` is considered deprecated in favor of `@QuarkusIntegrationTest` which provides a superset of the testing
+This procedure was formerly accomplished using the `@NativeImageTest` annotation. `@NativeImageTest` was replaced by `@QuarkusIntegrationTest` which provides a superset of the testing
```

Comment:
Not sure if this still worth keeping around.

Reply:
I kept it so people using `@NativeImageTest` in legacy references could understand why that annotation doesn't exist anymore

Reply:
Sure, but I would expect this to be necessary only during the deprecation face. Anyway, it's OK with me.